packages/jest-environment/src/index.ts

```diff
@@ -14,17 +14,19 @@ import {JestFakeTimers as FakeTimers} from '@jest/fake-timers';
 type JestMockFn = typeof jestMock.fn;
 type JestMockSpyOn = typeof jestMock.spyOn;
 
-export type EnvironmentContext = {
-  console?: Console;
-  testPath?: Config.Path;
-};
+// In Jest 25, remove `Partial` since it's incorrect. The properties are always
+// passed, or not. The context itself is optional, not properties within it.
+export type EnvironmentContext = Partial<{
+  console: Console;
+  docblockPragmas: {[key: string]: string | Array<string>};
```

Comment:
Is the docblock types exported from anywhere? Maybe not worth the dep

Reply:
Yeah, I didn’t think it was worth the dependency.